hieradata/vagrant_credentials.yaml

```diff
@@ -35,8 +35,6 @@ govuk::apps::content_audit_tool::db::password: 'MDgTmQKvEAax4wgefQAZRNDTajkdtEVf
 govuk::apps::content_audit_tool::db_password: "%{hiera('govuk::apps::content_audit_tool::db::password')}"
 govuk::apps::content_data_admin::db::password: 'FrkwKJGjLAjbznLtPhFjVy7YEonLKJTr'
 govuk::apps::content_data_admin::db_password: "%{hiera('govuk::apps::content_data_admin::db::password')}"
-govuk::apps::content_performance_manager::db_password: "%{hiera('govuk::apps::content_performance_manager::db::password')}"
-govuk::apps::content_performance_manager::db::password: 'jo6Kah0kuokaighoughePhooch1Eequu'
```

Comment:
no keys for content_data_api?

Reply:
I'm very unsure what (if anything) this vagrant_credentials.yaml file relates to... I'll investigate more though.

Reply:
So, I think this relates to the Vagrantfile at the root of the govuk-puppet Git repository. I've opened a Pull Request to update the comment in the file https://github.com/alphagov/govuk-puppet/pull/9096 and added a commit to this branch to add some credentials for the Content Data API just in case.